recipes/newrelic/infrastructure/network/ktranslate-syslog.yml

```diff
@@ -0,0 +1,285 @@
+# Visit our schema definition for additional information on this file format.
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+# Expected Variables:
+#   SERVICE_NAME - string value indicating syslog install
```

Comment:
> is it true that ktranslate is always running within a container
Nope. In fact, this recipe installs ktranslate as a service (just like infra agent) via a package manager, not as a container. 

It might work better to do a more thorough check like `SERVICE_NAME=syslog` vs just a blanket "_does `SERVICE_NAME` exist as an env var_". I can add that into the mix if it's helpful

Reply:
Got it.  I jumped to the container thought after speed reading our [docs re: ktranslate](https://docs.newrelic.com/docs/network-performance-monitoring/get-started/npm-introduction).

To be safe we may want to be a bit more thorough as you alluded if it's not too much trouble.  

PR looks good, pending @Julien4218 or @NRhzhao's input.